k8s/perftest/common/pod-identity/pod-identity.yaml

```diff
@@ -0,0 +1,205 @@
+apiVersion: v1
```

Comment:
I think we could just move this to common and bin the old AAT? 
@adusumillipraveen @luigibk ?

Reply:
Yes, agree for this.

Reply:
moved the pod identity to common